### PR TITLE
fix: addresses accessibility warning in expand/show more block and accordion block

### LINKED ISSUE #2617: Expand / Show More & Accordion block is causing an accessibility warning in PageSpeed insights
**Describe the bug**
- Related to the ff. issues: 
   - https://www.notion.so/Michael-Rowney-037d961136a54771b7a62998b88003ea
   - https://secure.helpscout.net/conversation/2264421730/20715?folderId=2526706
- Blocks affected:
   - Accordion
   - Expand / Show More
- Example of a page analysis with an expand / show more block in the page: https://pagespeed.web.dev/report?url=https%3A%2F%2Fsandbox.gambit.ph%2Fandrea%2Fexpand-show-more-aria-issue%2F

Note: Make sure to try this issue on Sandbox to recognize page in pagespeed insights website

**To Reproduce**
1. Add a new page
2. Add both blocks mentioned above
3. Publish page
4. Copy link of the published page
5. In https://pagespeed.web.dev/, paste the page and click Analyze
6. After the analysis has been finished, search "ARIA"
7. Open this dropdown
image
8. See Failing Elements. This is how the error would show:
_**Expand / Show More**_
image
**_Accordion_**
image

**Expected behavior**
It should not cause an accessibility issue for Expand / Show More block & Accordion block

**Screenshots**

https://user-images.githubusercontent.com/103395655/225233486-5668f8d5-657f-4470-88fe-ba15d5f479c6.mp4

### PR DESCRIPTION
**Pull request by mxkae**
fixes #2617 

- removed custom attributes aria-expanded in Expand / Show More block

### CODE REVIEW

**Comment by andeng1106** [comment]
@mxkae Issue has been fixed for v3 blocks 😁

Remaining issues:
1. V3 Expand / Show More causes a block error when updating build from 3.8.2 (latest) to PR build

- This only happens for existing/old block. New blocks added doesn't cause an issue.

https://github.com/gambitph/Stackable/assets/103395655/551cc09c-22fc-4dbb-975c-7892240e68ad

2. Fix is not yet applied for v2 Accordion & Expand / Show More blocks (for confirmation w/ @bfintal)

File: src/block/expand/schema.js [hunk 1]
```diff
@@ -30,9 +30,7 @@ export const attributes = ( version = VERSION ) => {
 	ConditionalDisplay.addAttributes( attrObject )
 
 	attrObject.addDefaultValues( {
-		attributes: {
-			customAttributes: [ [ 'aria-expanded', 'false' ] ],
-		},
+		attributes: {},
 		versionAdded: '3.0.0',
 		versionDeprecated: '',
```
Comment (bfintal) [inline src/block/expand/schema.js:35]:
```suggestion
		versionDeprecated: '3.9.0',
```

File: src/block/expand/edit.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ const TEMPLATE = [
 		text: __( 'Show more', i18n ),
 		linkUrl: '#',
 		className: 'is-style-link stk-block-expand__show-button',
-		customAttributes: [ [ 'aria-hidden', 'false' ], [ 'role', 'button' ] ],
+		customAttributes: [ [ 'aria-hidden', 'false' ], [ 'role', 'button' ], [ 'aria-expanded', 'true' ] ],
```
Comment (bfintal) [inline src/block/expand/edit.js:50]: You are going to click this button to "show more", so this should be `aria-expanded` `false`
```suggestion
		customAttributes: [ [ 'aria-hidden', 'false' ], [ 'role', 'button' ], [ 'aria-expanded', 'false' ] ],
```

File: src/block/expand/edit.js [hunk 3]
```diff
@@ -58,7 +58,7 @@ const TEMPLATE = [
 		text: __( 'Show less', i18n ),
 		linkUrl: '#',
 		className: 'is-style-link stk-block-expand__hide-button',
-		customAttributes: [ [ 'aria-hidden', 'true' ], [ 'role', 'button' ] ],
+		customAttributes: [ [ 'aria-hidden', 'true' ], [ 'role', 'button' ], [ 'aria-expanded', 'false' ] ],
```
Comment (bfintal) [inline src/block/expand/edit.js:61]: Just like the above, this should be `true`
```suggestion
		customAttributes: [ [ 'aria-hidden', 'true' ], [ 'role', 'button' ], [ 'aria-expanded', 'true' ] ],
```

File: src/block/expand/schema.js [hunk 4]
```diff
@@ -34,7 +34,7 @@ export const attributes = ( version = VERSION ) => {
 			customAttributes: [ [ 'aria-expanded', 'false' ] ],
 		},
 		versionAdded: '3.0.0',
-		versionDeprecated: '',
+		versionDeprecated: '3.9.0',
```
Comment (bfintal) [inline src/block/expand/schema.js:37]:
```suggestion
		versionDeprecated: '3.9.1',
```

**Review by bfintal — CHANGES_REQUESTED** [review]
Please see my comments

**Comment by andeng1106** [comment]
No block error anymore when updating builds.

Remaining issue:
- The ARIA warning for Expand / Show More block showed in PageSpeed Insights:
Screenshot 2023-06-16 at 10 37 30 AM
 
@mxkae

**Comment by bfintal** [comment]
@andeng1106 does this pagespeed issue appear with old blocks or newly created blocks? If it's old, we might need to add some deprecation to remove any existing `aria-expanded` custom attributes

**Comment by andeng1106** [comment]
> @andeng1106 does this pagespeed issue appear with old blocks or newly created blocks? If it's old, we might need to add some deprecation to remove any existing `aria-expanded` custom attributes

This happens with old blocks and newly created blocks. @bfintal

**Pull request merged**